staking/cli/src/instructions.rs

```diff
@@ -216,15 +226,18 @@ pub fn process_transaction(
             CommitmentConfig::confirmed(),
             RpcSendTransactionConfig {
                 skip_preflight: true,
+                max_retries: Some(10),
```

Comment:
I know it's counter intuitive but rpc providers usually recommend not setting this

Reply:
https://docs.triton.one/chains/solana/sending-txs

Reply:
fixed.